tsconfig.json

```diff
@@ -20,7 +20,7 @@
     // "composite": true,                           /* Enable project compilation */
     // "tsBuildInfoFile": "./",                     /* Specify file to store incremental compilation information */
     // "removeComments": true,                      /* Do not emit comments to output. */
-    // "noEmit": true,                              /* Do not emit outputs. */
+    "noEmit": true,                                 /* Do not emit outputs. */
```

Comment:
Without this, vscode appears to think it's going to overwrite js ontop of itself. It won't but it doesn't stop the problem output pane from crying about it.